sql/src/main/java/org/apache/druid/sql/calcite/schema/MetadataSegmentView.java

```diff
@@ -106,10 +106,15 @@ public void start()
     }
     try {
       if (isCacheEnabled) {
-        scheduledExec.schedule(new PollTask(), 0, TimeUnit.MILLISECONDS);
-        lifecycleLock.started();
-        log.info("MetadataSegmentView Started.");
+        try {
+          poll();
+        }
+        finally {
```

Comment:
Please add a catch block to emit exceptions.

Reply:
added catch block